contentcuration/contentcuration/frontend/accounts/pages/Main.vue

```diff
@@ -95,8 +100,24 @@
         showTermsOfService: false,
       };
     },
+    computed: {
+      searchParams() {
+        return new URLSearchParams(window.location.search.substring(1));
```

Comment:
@jayoshih I changed this to use the `search` property so it won't pick up the URL fragment. Let me know if that's an issue.

Reply:
So turns out the way the test was testing this was actually only changing the URL fragment and not the query string.